hack/dockerfile/install/vndr.installer

```diff
@@ -1,11 +1,12 @@
 #!/bin/sh
 
-: ${VNDR_COMMIT:=f5ab8fc5fb64d66b5c6e55a0bcb58b2e92362fa0}
-
 install_vndr() {
+	: "${VNDR_COMMIT?}"
+	: "${PREFIX?}"
+
 	echo "Install vndr version $VNDR_COMMIT"
 	git clone https://github.com/LK4D4/vndr.git "$GOPATH/src/github.com/LK4D4/vndr"
-	cd "$GOPATH/src/github.com/LK4D4/vndr"
+	cd "$GOPATH/src/github.com/LK4D4/vndr" || return
```

Comment:
There's no need to have this `|| return`, since we do `set -e` in `hack/dockerfile/install/install.sh`.

Same in other places.

Reply:
Ah, right; so my linter highlights it, because (obviously) it doesn't know it's being sourced by that file. Perhaps better to repeat the `set -e` in this file, for situations where it's not called through `install.sh` but directly